ios/RNSScreen.h

```diff
@@ -85,9 +85,11 @@ namespace react = facebook::react;
 // Props controlling UISheetPresentationController
 @property (nonatomic) RNSScreenDetentType sheetAllowedDetents;
 @property (nonatomic) RNSScreenDetentType sheetLargestUndimmedDetent;
+@property (nonatomic) NSNumber *sheetCustomLargestUndimmedDetent;
 @property (nonatomic) BOOL sheetGrabberVisible;
 @property (nonatomic) CGFloat sheetCornerRadius;
 @property (nonatomic) BOOL sheetExpandsWhenScrolledToEdge;
+@property (nonatomic) NSArray<NSNumber *> *sheetCustomDetents;
```

Comment:
`sheetDetentValues`, `sheetMaxDetentValueFractions` or some variation of these?

Reply:
I would stay with `sheetCustomDetents`, as `sheetAllowedDetents` is already there and it would be confusing to use just`sheetDetentValues`. Also, `sheetMaxDetentValueFractions` seems a bit odd for me 🤔